openff/toolkit/typing/engines/smirnoff/forcefield.py

```diff
@@ -98,7 +98,7 @@ def _get_installed_offxml_dir_paths() -> list[str]:
 
 
 def get_available_force_fields(full_paths=False):
-    """
```

Comment:
This is the only auto-magic change I'm iffy on, and that's mostly because I didn't build docs locally

Reply:
The docstring of this one looks okay

<img width="1138" alt="image" src="https://github.com/user-attachments/assets/525fde94-7ab3-440d-80f4-8dfad556f440">